### ISSUE TITLE
[Bug]: Cluster do not see slave nodes 

### ISSUE THREAD

**Issue by jakub-kubera** [RedisInsight/RedisInsight#3035]
I have cluster with master nodes and slave nodes, replicas, but when I have added master nodes it showing only master nodes

**Comment by jakub-kubera** [comment]
I do not want to monitor replicas, but it would be good to know which replicas are in cluster, how many replicas are there, if slaves are in sync, possibility to failover if needed, resync keys, promote slave to master, move shards, make reshard, currently there is no any additional value/option with redisinsight, only checking cached key values

**Comment by Skoucail** [comment]
I totally agree, i used the failover feature in redisinsight v1 a lot. (fe when doing controlled rolling upgrades of the cluster)
And having a visual representation of the cluster with masters and replica's was a quick wat to see that everything was looking ok.

Also because Workbench/CLI doesn't have the feature to select the endpoint like we were able in v1. Its now almost impossible to trigger failovers with redis insight. (#2809)

**Comment by Skoucail** [comment]
I agree, having a complete view on the cluster is a plus.
But in v1 the function i use the most (after the browser ofc) is the failover feature.
I do i all the time when upgrading my cluster so i only take down nodes that are replica's.

So i would love to see this feature coming back in v2
